src/evaluate/evaluate.ts

```diff
@@ -42,9 +44,20 @@ export class Evaluator {
       this.context.addReference(new SimpleReference(paramName));
     });
 
-    return this.context.template.resources.forEach((_logicalId, resource) =>
+    this.context.template.resources.forEach((_logicalId, resource) =>
       this.evaluateResource(resource)
     );
+
+    this.context.template.outputs.forEach((output, outputId) => {
+      const outputValue = this.evaluate(output.value);
+      new DeCDKCfnOutput(this.context.stack, outputId, {
```

Comment:
Yes, that's clever and works well! I like it.

What you might have thought is that intrinsics are allowed in the value (and exportName) - but CDK expects them to be a `string`.  However in CDK intsinics are actually modelled as Tokens, which behave like strings. So I suspect that using the CDK class would work as well. To make sure we only get supported/expected values we would add checks to the parser and/or type resolutions. We might have to make typescript happy, but that easy enough. Not sure if there's any benfit to use the CDK class.

Reply:
I see. Thanks for the insight!

If CDK token works in this case then I feel it would be better to move to the CDK class eventually for less maintenance work in future. But I think we can keep this class in the interim so it accepts condition as a `string`. 

I think this can be included as an investigation in the issue where we change condition from `string` to `CfnCondition`.